homework.rb

```diff
@@ -1,6 +1,14 @@
-VARIANT = 0
+VARIANT = 1
 
-def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+
+def random_4_numbers 
+      rand(1000..10000)
+end
+
+def generate 
+  first_random_four_numbers = random_4_numbers
+  second_random_four_numbers = random_4_numbers
+  third_random_four_numbers = random_4_numbers
```

Comment:
prev 3 lines are not necessary
you can just call random_4_numbers method in string with interpolation